src/main/kotlin/calculator/Player.kt

```diff
@@ -0,0 +1,18 @@
+package calculator
+
+fun main() {
+    val calculator = StringCalculator()
+    while (true) {
+        print("계산할 문자열을 입력해주세요. ('exit'을 입력하면 종료됩니다): ")
+        val input = readLine()
```

Comment:
개인적으로 nullable한 타입을 쓰는걸 좋아하지 않는데요. 😃 (null 처리에 따른 복잡성 추가.. 등등)
가급적 non-nullable한 타입이 되도록 앞단에서 만들어주면 어떨까요? 

```suggestion
        val input = readlnOrNull() ?: throw IllegalArgumentException("...")
```

1.6.0 버전 이후부터는 readLine() 대신에 [readlnOrNull() 메서드를 사용](https://lovelyunsh.tistory.com/284)해야한다고 하네요. 😃

Reply:
네! 저도 고민한 부분인데요, `InputValidator`가 input을 검증하는 로직이다보니,
검증 행위를 해당 객체에 응집도있게 뭉치려고 했어요.

물론 그러다보니 nullable한 상태가 해당 객체까지 흘러들어가는 문제?가 발생했는데,
말씀 주신것처럼 앞단에서 처리하는 것도 적절해보이네요! 수정하도록 하겠습니다 🙇‍♂️